dbms/src/AggregateFunctions/AggregateFunctionSum.cpp

```diff
@@ -91,14 +104,13 @@ AggregateFunctionPtr createAggregateFunctionSum(const std::string & name, const
 
 } // namespace
 
-extern const String count_second_stage = NameCountSecondStage::name;
-
 void registerAggregateFunctionSum(AggregateFunctionFactory & factory)
 {
-    factory.registerFunction("sum", createAggregateFunctionSum<AggregateFunctionSumSimple>, AggregateFunctionFactory::CaseInsensitive);
-    factory.registerFunction("sumWithOverflow", createAggregateFunctionSum<AggregateFunctionSumWithOverflow>);
-    factory.registerFunction("sumKahan", createAggregateFunctionSum<AggregateFunctionSumKahan>);
-    factory.registerFunction(count_second_stage, createAggregateFunctionSum<AggregateFunctionCountSecondStage>, AggregateFunctionFactory::CaseInsensitive);
+    factory.registerFunction(NameSum::name, createAggregateFunctionSum<AggregateFunctionSumSimple, NameSum>, AggregateFunctionFactory::CaseInsensitive);
+    factory.registerFunction(NameSumWithOverFlow::name, createAggregateFunctionSum<AggregateFunctionSumWithOverflow, NameSumWithOverFlow>);
+    factory.registerFunction(NameSumOnPartialResult::name, createAggregateFunctionSum<AggregateFunctionSumOnPartialResultStage, NameSumOnPartialResult>);
```

Comment:
Since `NameSumOnPartialResult` is the same as `NameSumWithOverFlow`, then `SumWithOverFlow` has been registered twice?

Reply:
> Since `NameSumOnPartialResult` is the same as `NameSumWithOverFlow`, then `SumWithOverFlow` has been registered twice?

I will delete one of them